addons-example.yml

```diff
@@ -37,6 +36,10 @@ my-s3-bucket:
     production:
       bucket_name: my-bucket-prod
       deletion_policy: Retain
+      retention_policy: # Optional
+        mode: GOVERNANCE # or COMPLIANCE
```

Comment:
What is the difference? Is that documented somewhere?

Reply:
Governance means the bucket objects can be modified by an admin with the appropriate perms. Compliance means not even admins can remove or modify objects in the bucket.

Reply:
FYI: https://aws.amazon.com/blogs/storage/protecting-data-with-amazon-s3-object-lock/

Reply:
I think this might be helpful...
```suggestion
        mode: GOVERNANCE # or COMPLIANCE (See https://aws.amazon.com/blogs/storage/protecting-data-with-amazon-s3-object-lock/)
```